onnxconverter_common/auto_float16.py

```diff
@@ -0,0 +1,184 @@
+import onnxruntime as ort
+import onnx
+import numpy as np
+from onnxconverter_common import float16
+from onnx import helper, mapping
+import copy
+
+
+def main():
+    # Fill in this template to convert a model, or call auto_convert_float_to_float16 from another script
+    model_path = "<model path here>"
+    model = onnx.load(model_path)
+    test_data = {"<input1 name>": np.array(), "<input2 name>": np.array()}
+
+    def validate(res1, res2):
```

Comment:
nit: maybe could have a static function in this file to generate this functor with customized atol/rtol